src/System.IO.FileSystem/tests/Performance/System.IO.FileSystem.Performance.Tests.csproj

```diff
@@ -37,5 +37,8 @@
       <Name>PerfRunner</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
No idea why VS keeps wanting to add this. Assuming it is a standard thing now?

Reply:
Yes, this is annoying, anyway, I have been avoiding merging those.

Reply:
From searching VS sources, it appears to be written for "StartupTestService" which automatically runs at project load. Possibly related to unit testing? Seems like a bug, but harmless.

Reply:
Interestingly tests are now being discovered in the TestExplorer. They don't run successfully, but we may be much closer to actually working in VS...

```
[3/9/2018 3:49:56 PM Informational] ------ Load Playlist started ------
[3/9/2018 3:49:56 PM Informational] ========== Load Playlist finished (0:00:00) ==========
[3/9/2018 4:06:54 PM Informational] ------ Load Playlist started ------
[3/9/2018 4:06:54 PM Informational] ========== Load Playlist finished (0:00:00) ==========
[3/9/2018 4:26:37 PM Informational] ------ Run test started ------
[3/9/2018 4:26:42 PM Warning] No test is available in N:\repos\corefx\bin\Windows_NT.AnyCPU.Debug\System.IO.FileSystem.Tests\netcoreapp\System.IO.FileSystem.Tests.dll. Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.
[3/9/2018 4:26:42 PM Informational] ========== Run test finished: 0 run (0:00:05.6674206) ==========
```

Reply:
I'd opened a bug about it.  I'm told it's being fixed.